tests/stdlib/tioutils.nim

```diff
@@ -0,0 +1,49 @@
+discard """
+  output: '''
+hello1
+hello1
+  '''
+"""
+
+import std/ioutils
+import os
+
+block:
+  let tmpFileName = "./tmpFile.txt"
```

Comment:
* too generic, could potentially clash with other temp files from other tests in the future (tests can run in parallel), either use a uuid or name it `tioutils.txt`, which also makes its association with this file more greppable.
* nit: in some tests I'm using `from stdtest/specialpaths import buildDir` (which avoids polluting sources with tmp files, eg in case of a crash before tmp file collection which makes git status dirty), but feel free to ignore since it's not (yet?) common practice